crates/table/src/table.rs

```diff
@@ -486,10 +486,12 @@ impl Table {
     /// On `Ok(_)`, statistics of the table are also updated,
     /// and the `ptr` still points to a valid row, and otherwise not.
     ///
+    /// If `CHECK_SAME_ROW` holds, an identical row will be treated as a set-semantic duplicate.
```

Comment:
If `CHECK_SAME_ROW` is false, what becomes of identical rows? Are they ignored? Do they signal some different error?